Makefile

```diff
@@ -170,6 +170,10 @@ docker_push: docker_tag docker_auth ## push to docker hub
 lint-imports:
 	lint-imports
 
+migrate:
+	python manage.py lms migrate --database default --noinput --traceback --pythonpath=.
+	python manage.py cms migrate --database default --noinput --traceback --pythonpath=.
```

Comment:
Were these options copied from provisioning scripts or some other location? This is probably correct, but I don't know enough about the arguments, and it would be nice to see that it just matches some other case.

Reply:
I did grab these from the `make dev.migrate.lms/cms` commands, but I didn't specify the settings file: https://github.com/openedx/devstack/blob/master/Makefile#L257-L263